app/jobs/clock/subscriptions_to_be_terminated_job.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Clock
+  class SubscriptionsToBeTerminatedJob < ApplicationJob
+    queue_as 'clock'
+
+    def perform
+      Subscription
+        .joins(customer: :organization)
+        .joins('left join webhooks on subscriptions.id = webhooks.object_id and '\
+               "webhooks.webhook_type = 'alert.subscription_to_be_terminated' and webhooks.status = 1"
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)